tests/Annotations/Serializer/HypothesisClientAnnotationNormalizerApplicationTest.php

```diff
@@ -738,6 +728,135 @@ public function normalizeProvider() : array
         ];
     }
 
+    /**
+     * @test
+     * @dataProvider sanitizeProvider
+     */
+    public function it_will_sanitize_annotations(array $expected, Annotation $annotation)
+    {
+        $this->assertEquals($expected, $this->normalizer->normalize($annotation));
+    }
+
+    public function sanitizeProvider() : array
+    {
+        $createdDate = '2017-11-29T17:41:28Z';
+
+        return [
+            'script-tags' => [
+                [
+                    'id' => 'id',
+                    'access' => 'public',
+                    'content' => [
+                        [
+                            'type' => 'paragraph',
+                            'text' => 'NOTE: It is not possible to display this content.',
+                        ],
+                    ],
+                    'created' => $createdDate,
+                    'document' => [
+                        'title' => 'title',
+                        'uri' => 'uri',
+                    ],
+                ],
+                new Annotation(
+                    'id',
+                    '<script>evil()</script>',
```

Comment:
Since the code for purification etc has been extracted, these should be unit tests on the relevant renderer.

(Since this is important I'm happy with a single high-level test that also checks that it's configured correctly, but that doesn't belong here.)